website/lib/build.js

```diff
@@ -44,7 +44,7 @@ if (process.env.NETLIFY_API_KEY) {
 
 // use KaTeX from CDN on the main page for Netlify production deploy
 if (process.env.CONTEXT === 'production') {
-    const version = require('../versions.json')[0];
+    const {version} = require('../../package.json');
```

Comment:
We used to set the `version` to the next version `-pre`. Is this no longer the case?  (It seems not, but wanted to check.)  Otherwise LGTM.